public/stylesheets/overlays/generic/generic.css

```diff
@@ -0,0 +1,647 @@
+        /* Commentator cam */ 1910px 710px, 1910px 365px, 1280px 365px, 
+        /* Cam separator */ 1280px 355px, 
+        /* Player cam */ 1910px 355px, 1910px 10px, 1280px 10px, 1280px 710px, 1270px 710px,
+        /* Right game edge */ 1270px 10px, 
+        1130px 10px, 1122px 30px, 870px 30px, 
+        850px 60px, 430px 60px, 410px 30px, 
+        158px 30px, 150px 10px, 10px 10px
+    );
+}
+
+.container {
+    width: 1920px;
+    height: 1080px;
+    position: absolute;
+    left: 0;
+    top: 0;
+}
+
+#background {
+    background-image: url(http://i.imgur.com/rL7QNS0.jpg);  
+}
+
+#main-container {
+    -webkit-clip-path: none;
+}
+
+/* empty-layout */
+
+    #main-container.empty-layout .playerText {
+        position: absolute;
+        color: white;
+        font-size: 42px;
+        font-family: 'Oswald', cursive;
+        text-shadow: 2px 2px 2px rgb(50, 50, 50);
+
+        /* Oswald sits a little low, so raise the baseline */
+        margin-top: -5px;
+
+        overflow: hidden;
+        overflow-wrap: break-word;
+    }
+
+    #main-container.empty-layout .title,
+    #main-container.empty-layout .tourneyInfo,
+    #main-container.empty-layout .commentators,
+    #main-container.empty-layout .lCharacter,
+    #main-container.empty-layout .rCharacter,
+    #main-container.empty-layout .lIcons,
+    #main-container.empty-layout .lState, 
+    #main-container.empty-layout .lPort,    
+    #main-container.empty-layout .lPlayer, 
+    #main-container.empty-layout .lScore, 
+    #main-container.empty-layout .rScore, 
+    #main-container.empty-layout .rPlayer, 
+    #main-container.empty-layout .rPort, 
+    #main-container.empty-layout .rState, 
+    #main-container.empty-layout .rIcons  {
+        display: none;
+    }
+
+/* player-layout */
+
+    #main-container.player-layout .playerText {
+        position: absolute;
+        color: white;
+        font-size: 42px;
+        font-family: 'Oswald', cursive;
+        text-shadow: 2px 2px 2px rgb(50, 50, 50);
+
+        /* Oswald sits a little low, so raise the baseline */
+        margin-top: -5px;
+
+        overflow: hidden;
+        overflow-wrap: break-word;
+    }
+
+    #main-container.player-layout .title {
+        top: 0px;
+        width: 420px;
+        left: 750px;
+        height: 60px;
+        text-align: center;
+    }
+
+    #main-container.player-layout .tourneyInfo,
+    #main-container.player-layout .commentators,
+    #main-container.player-layout .lCharacter,
+    #main-container.player-layout .rCharacter,
+    #main-container.player-layout .lIcons,
+    #main-container.player-layout .lState, 
+    #main-container.player-layout .lPort,   
+    #main-container.player-layout .lPlayer, 
+    #main-container.player-layout .lScore, 
+    #main-container.player-layout .rScore, 
+    #main-container.player-layout .rPlayer, 
+    #main-container.player-layout .rPort, 
+    #main-container.player-layout .rState, 
+    #main-container.player-layout .rIcons  {
```

Comment:
This seems like a duplicate CSS rule to the `#main-container.empty-layout` stuff, so you could combine them.